{"Timestamp":"2/22/2017 7:32:35 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 10:12:25 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:05:30 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:07:24 AM","Message":"Exception has been thrown by the target of an invocation.","InnerEx":"Failed to compare two elements in the array.","Source":"mscorlib","StackTrace":"   at InstaTech.App_Code.Socket_Handlers.Socket_Main.OnMessage(String message) in C:\\Users\\Typic\\Documents\\Source\\WebSites\\InstaTech_Server\\InstaTech_Server\\App_Code\\Socket_Handlers\\Socket_Main.cs:line 147\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:08:35 AM","Message":"Exception has been thrown by the target of an invocation.","InnerEx":"Failed to compare two elements in the array.","Source":"mscorlib","StackTrace":"   at InstaTech.App_Code.Socket_Handlers.Socket_Main.OnMessage(String message) in C:\\Users\\Typic\\Documents\\Source\\WebSites\\InstaTech_Server\\InstaTech_Server\\App_Code\\Socket_Handlers\\Socket_Main.cs:line 147\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:17:39 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:24:49 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 11:56:50 AM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 12:07:50 PM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 12:21:49 PM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 12:53:23 PM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
{"Timestamp":"2/22/2017 1:11:12 PM","Message":"The I/O operation has been aborted because of either a thread exit or an application request","InnerEx":null,"Source":"System.Web","StackTrace":"   at System.Web.WebSockets.WebSocketPipe.\u003c\u003ec__DisplayClass9_0.\u003cReadFragmentAsync\u003eb__0(Int32 hrError, Int32 cbIO, Boolean fUtf8Encoded, Boolean fFinalFragment, Boolean fClose)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003cDoWork\u003ed__45`1.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Web.WebSockets.AspNetWebSocket.\u003c\u003ec__DisplayClass36_0.\u003c\u003cReceiveAsyncImpl\u003eb__0\u003ed.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketMessageReader.\u003cReadMessageAsync\u003ed__0.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.Web.WebSockets.WebSocketHandler.\u003cProcessWebSocketRequestAsync\u003ed__9.MoveNext()"}
